test-data/unit/deps-generics.test

```diff
@@ -140,5 +140,24 @@ def f(x: T) -> None:
 [out]
 <m.A.__add__> -> m.f
 <m.A.f> -> m.f
-<m.A> -> <m.T>, m.A
+<m.A> -> <m.T>, <m.f>, m.A, m.f
+<m.T> -> <m.f>, m.f
+
+[case testTypeVarValues]
```

Comment:
Also test a generic class where the class type variable has values and the type variable is used in a method.